sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * soc-apci-intel-adl-match.c - tables and support for ADL ACPI enumeration.
+ *
+ * Copyright (c) 2020, Intel Corporation.
+ *
+ */
+
+#include <sound/soc-acpi.h>
+#include <sound/soc-acpi-intel-match.h>
+
+static const struct snd_soc_acpi_endpoint single_endpoint = {
+	.num = 0,
+	.aggregated = 0,
+	.group_position = 0,
+	.group_id = 0,
+};
```

Comment:
hm, is this because other files do the same? That's 4 redundant statements right there... That could've been just `static const struct snd_soc_acpi_endpoint single_endpoint;`

Reply:
@lyakh yes, I faintly remember we've had this discussion before and @plbossart prefers to have these explicit.